caffe2/utils/python_bindings.cc

```diff
@@ -0,0 +1,13 @@
+#include "caffe2/utils/signal_handler.h"
+#include <torch/csrc/utils/pybind.h>
+
+namespace py = pybind11;
+
+namespace caffe2 {
+
+PYBIND11_MODULE(python_bindings, m) {
+  m.def("set_print_stack_traces_on_fatal_signal",
```

Comment:
I wonder if it's worth exposing something like this more generally in PyTorch distributed/pytorch. It seems pretty useful for developers debugging their local changes in addition to how it's used for unittests here.